ui/src/literature/components/LiteratureItem.jsx

```diff
@@ -148,6 +150,13 @@ function LiteratureItem({ metadata, searchRank }) {
             {eprints && <ArxivEprintList eprints={eprints} />}
           </InlineUL>
         </div>
+        {canEdit && (
```

Comment:
I would explicit check the roles, in the future `canEdit` might be the submitter as well and we don't want to display the pub notes